Sources/GIF/code/GIFEncoder.swift

```diff
@@ -2,6 +2,8 @@ import Foundation
 import Logging
 import CairoGraphics
 import Utils
+import QuartzCore
+import System
```

Comment:
I used QuartzCore for `CACurrentMediaTime()` to test whether optimizations helped or harmed performance. I couldn't detect a measurable difference in the optimization that fuses loop iterations for bit-by-bit encoding. Theoretically it would improve performance, not harm performance, by reducing the total number of instructions issued.